packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -865,59 +877,92 @@ export class GarbageCollector implements IGarbageCollector {
     }
 
     /**
-     * Validates that all new references are correctly identified and processed. The basic principle for validation is
-     * that we should not have new references in the reference graph (GC data) that have not been notified to the
-     * garbage collector via `referenceAdded`.
-     * We validate that the references in the current reference graph should be a subset of the references in the last
-     * run's reference graph + references since the last run.
+     * Finds Missing Routes and logs a Performance event everytime a missing route is found
+     *
+     * Expected Routes = Routes in Current datastore graph and not in Previous datastore graph
+     * New Routes = New routes added by GC when a handle was added or root datastores
+     * Missing Routes = Expected Routes not in New Routes
+     *
+     * Node - represented as nodeId, it's a node on the GC graph
+     * Route - a reference with a parent, think `nodeId` & `reference`
```

Comment:
The definitions of route, reference and outbound is not correct. Infact, I think its incorrectly used in a lot of places in this file and in other places.
An "outbound route" and a "reference" mean the same thing. They represent a link from one node to another. For example, consider the following reference graph: A -> B -> C. Here, the arrows represent reference or outbound route. Basically, there is a reference or outbound route from A to B and from B to C. So, essentially the source node, destination node and the link between together mean a reference or an outbound route.